src/i18n/resources/fr/fra.js

```diff
@@ -167,6 +217,11 @@ module.exports = {
       low: 'Autres sources de données – par exemple, estimations d’experts ou estimations reposant sur des évaluations non compatibles.',
     },
   },
+  footer1:
+    '¹ Données ne remontant pas à plus de 5 ans à compter de l’année de soumission du rapport (2018 ou plus récent pour les rapports nationaux de FRA 2025)',
+  footer2:
+    '² Données remontant à plus de 5 ans à partir de l’année de soumission du rapport (antérieures à 2018 pour les rapports nationaux de FRA 2025)',
+  footer3: '³ Compatibles en termes de méthodes, de catégories et de définitions utilisées',
```

Comment:
1. Données ne remontant pas à plus de 5 ans à compter de l’année de soumission du rapport (2018 ou plus récent pour les rapports nationaux de FRA 2025)

2. Données remontant à plus de 5 ans à partir de l’année de soumission du rapport (antérieures à 2018 pour les rapports nationaux de FRA 2025)

3. Compatibles en termes de méthodes, de catégories et de définitions utilisées

 

1. Données ne remontant pas à plus de 10 ans à compter de l’année de soumission du rapport (2013 ou plus récent pour les rapports nationaux de FRA 2025)

2. Données remontant à plus de 10 ans à compter de l’année de soumission du rapport (antérieures à 2013 pour les rapports nationaux de FRA 2025)


The other footer is missing (see eng)

Reply:
it's a "part of" :)

<img width="1111" alt="Screenshot 2023-06-09 alle 20 52 31" src="https://github.com/openforis/fra-platform/assets/10047945/268cf8d0-3e58-4c44-a407-3127797c3cac">